fuzz/fuzz_targets/http_api.rs

```diff
@@ -197,6 +197,7 @@ impl RequestHandler for StubApiRequestHandler {
                 landlock_rules: None,
                 #[cfg(feature = "ivshmem")]
                 ivshmem: None,
+                external_fds: None,
```

Comment:
not about this line but there is a typo in the commit message: I think UDS is not a common appreviation for Unix domain sockets? Could you please be a little more verbose here and write something like:

`FDs are passed as an array of .... via UNIX Domain Sockets using the SCM_RIGHTS mechanism.` or so?